helix-term/src/application.rs

```diff
@@ -1029,13 +1041,67 @@ impl Application {
                             .collect();
                         Ok(json!(result))
                     }
-                };
+                    MethodCall::ShowDocument(params) => {
+                        use helix_view::editor::Action;
+
+                        if params.external.unwrap_or(false) {
+                            match open::that(params.uri.as_str()) {
```

Comment:
`open::that` is a blocking function that is ill suited for use in async code (or really helix in general). See discussion in https://github.com/Byron/open-rs/pull/64 and https://github.com/helix-editor/helix/pull/5820

Reply:
I removed the commit. I guess the best thing is to leave this unimplemented for now or wait for Byron/open-rs#64?